baseclasses/testing/pyRegTest.py

```diff
@@ -265,7 +266,22 @@ def assert_allclose(self, actual, reference, name, rtol, atol, full_name=None):
         if full_name is None:
             full_name = name
         msg = f"Failed value for: {full_name}"
-        numpy.testing.assert_allclose(actual, reference, rtol=rtol, atol=atol, err_msg=msg)
+        # special case if we're comparing list of strings
+        if isinstance(reference, list) and isinstance(reference[0], str):
+            self.assert_equal(actual, reference, name)
+        else:
+            numpy.testing.assert_allclose(actual, reference, rtol=rtol, atol=atol, err_msg=msg)
+
+    def assert_equal(self, actual, reference, name):
+        if isinstance(reference, (list, tuple)):
+            for i, j in zip(actual, reference):
+                # cast both to tuple
+                if tuple(i) != tuple(j):
```

Comment:
Why do these need to be cast into tuples?

Reply:
This used to be necessary, but because we now
1. check for both `(list, tuple)`
2. we compare elements directly via `zip`

this can be removed. I will push that up.